feat: implement auto for handling releases

tinytim84 opened the pull request.

dependabot-preview[bot] committed:
chore(deps-dev): bump auto from 9.57.0 to 9.58.0

Bumps [auto](https://github.com/intuit/auto) from 9.57.0 to 9.58.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.57.0...v9.58.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps): bump eslint-plugin-jsdoc from 30.6.4 to 30.6.5

Bumps [eslint-plugin-jsdoc](https://github.com/gajus/eslint-plugin-jsdoc) from 30.6.4 to 30.6.5.
- [Release notes](https://github.com/gajus/eslint-plugin-jsdoc/releases)
- [Commits](https://github.com/gajus/eslint-plugin-jsdoc/compare/v30.6.4...v30.6.5)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

AndrewLeedham committed:
Merge pull request #418 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-plugin-jsdoc-30.6.5

chore(deps): bump eslint-plugin-jsdoc from 30.6.4 to 30.6.5

AndrewLeedham committed:
Merge pull request #416 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-9.58.0

chore(deps-dev): bump auto from 9.57.0 to 9.58.0

dependabot-preview[bot] committed:
chore(deps-dev): bump @auto-it/slack from 9.57.0 to 9.58.0

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 9.57.0 to 9.58.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.57.0...v9.58.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

AndrewLeedham committed:
Merge pull request #417 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-9.58.0

chore(deps-dev): bump @auto-it/slack from 9.57.0 to 9.58.0

tinytim84 committed:
feat: try dry-run

tinytim84 committed:
fix: just do git command

tinytim84 committed:
fix: syntax

tinytim84 committed:
fix: syntax

tinytim84 committed:
fix: update remote url

tinytim84 committed:
fix: try a git push

tinytim84 committed:
fix: try shipit dry-run

tinytim84 committed:
feat: add plugin to check remote path

tinytim84 committed:
fix: try and set auto.remote

dependabot-preview[bot] committed:
chore(deps-dev): bump react from 16.13.1 to 16.14.0

Bumps [react](https://github.com/facebook/react/tree/HEAD/packages/react) from 16.13.1 to 16.14.0.
- [Release notes](https://github.com/facebook/react/releases)
- [Changelog](https://github.com/facebook/react/blob/master/CHANGELOG.md)
- [Commits](https://github.com/facebook/react/commits/v16.14.0/packages/react)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps-dev): bump @auto-it/slack from 9.58.0 to 9.59.0

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 9.58.0 to 9.59.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.58.0...v9.59.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

AndrewLeedham committed:
Merge pull request #421 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-9.59.0

chore(deps-dev): bump @auto-it/slack from 9.58.0 to 9.59.0

AndrewLeedham committed:
Merge pull request #420 from xeroxinteractive/dependabot/npm_and_yarn/next/react-16.14.0

chore(deps-dev): bump react from 16.13.1 to 16.14.0

AndrewLeedham committed:
chore: move to v2 dependabot

AndrewLeedham committed:
chore: remove v1 dependabot

AndrewLeedham committed:
chore: fix v2 syntax

AndrewLeedham committed:
chore: fix dependabot syntax

AndrewLeedham committed:
chore: fix allow dependabot syntax

dependabot[bot] committed:
chore(deps-dev): bump semantic-release-plus from 18.2.2 to 18.2.3

Bumps [semantic-release-plus](https://github.com/semantic-release-plus/semantic-release) from 18.2.2 to 18.2.3.
- [Release notes](https://github.com/semantic-release-plus/semantic-release/releases)
- [Commits](https://github.com/semantic-release-plus/semantic-release/compare/v18.2.2...v18.2.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump @types/react from 16.9.52 to 16.9.53

Bumps [@types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react) from 16.9.52 to 16.9.53.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump eslint-config-prettier from 6.12.0 to 6.13.0

Bumps [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) from 6.12.0 to 6.13.0.
- [Release notes](https://github.com/prettier/eslint-config-prettier/releases)
- [Changelog](https://github.com/prettier/eslint-config-prettier/blob/master/CHANGELOG.md)
- [Commits](https://github.com/prettier/eslint-config-prettier/compare/v6.12.0...v6.13.0)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #427 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-config-prettier-6.13.0

chore(deps): bump eslint-config-prettier from 6.12.0 to 6.13.0

dependabot[bot] committed:
chore(deps): bump npm-user-validate from 1.0.0 to 1.0.1

Bumps [npm-user-validate](https://github.com/npm/npm-user-validate) from 1.0.0 to 1.0.1.
- [Release notes](https://github.com/npm/npm-user-validate/releases)
- [Commits](https://github.com/npm/npm-user-validate/compare/v1.0.0...v1.0.1)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #426 from xeroxinteractive/dependabot/npm_and_yarn/next/types/react-16.9.53

chore(deps-dev): bump @types/react from 16.9.52 to 16.9.53

AndrewLeedham committed:
Merge pull request #428 from xeroxinteractive/dependabot/npm_and_yarn/npm-user-validate-1.0.1

chore(deps): bump npm-user-validate from 1.0.0 to 1.0.1

dependabot[bot] committed:
chore(deps): bump eslint-plugin-react from 7.21.4 to 7.21.5

Bumps [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react) from 7.21.4 to 7.21.5.
- [Release notes](https://github.com/yannickcr/eslint-plugin-react/releases)
- [Changelog](https://github.com/yannickcr/eslint-plugin-react/blob/master/CHANGELOG.md)
- [Commits](https://github.com/yannickcr/eslint-plugin-react/compare/v7.21.4...v7.21.5)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump @typescript-eslint/eslint-plugin from 4.4.1 to 4.5.0

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 4.4.1 to 4.5.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.5.0/packages/eslint-plugin)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #429 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-plugin-react-7.21.5

chore(deps): bump eslint-plugin-react from 7.21.4 to 7.21.5

AndrewLeedham committed:
Merge pull request #430 from xeroxinteractive/dependabot/npm_and_yarn/next/typescript-eslint/eslint-plugin-4.5.0

chore(deps): bump @typescript-eslint/eslint-plugin from 4.4.1 to 4.5.0

tinytim84 committed:
fix: try checkout with GITHUB_TOKEN

AndrewLeedham committed:
Merge pull request #425 from xeroxinteractive/dependabot/npm_and_yarn/next/semantic-release-plus-18.2.3

chore(deps-dev): bump semantic-release-plus from 18.2.2 to 18.2.3

dependabot[bot] committed:
chore(deps): bump eslint-config-prettier from 6.13.0 to 6.14.0

Bumps [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) from 6.13.0 to 6.14.0.
- [Release notes](https://github.com/prettier/eslint-config-prettier/releases)
- [Changelog](https://github.com/prettier/eslint-config-prettier/blob/master/CHANGELOG.md)
- [Commits](https://github.com/prettier/eslint-config-prettier/compare/v6.13.0...v6.14.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump @typescript-eslint/parser from 4.4.1 to 4.5.0

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 4.4.1 to 4.5.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.5.0/packages/parser)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump jest from 26.5.3 to 26.6.0

Bumps [jest](https://github.com/facebook/jest) from 26.5.3 to 26.6.0.
- [Release notes](https://github.com/facebook/jest/releases)
- [Changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md)
- [Commits](https://github.com/facebook/jest/compare/v26.5.3...v26.6.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump auto from 9.58.0 to 9.60.0

Bumps [auto](https://github.com/intuit/auto) from 9.58.0 to 9.60.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.58.0...v9.60.0)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #435 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-9.60.0

chore(deps-dev): bump auto from 9.58.0 to 9.60.0

AndrewLeedham committed:
Merge pull request #433 from xeroxinteractive/dependabot/npm_and_yarn/next/jest-26.6.0

chore(deps-dev): bump jest from 26.5.3 to 26.6.0

AndrewLeedham committed:
Merge pull request #432 from xeroxinteractive/dependabot/npm_and_yarn/next/typescript-eslint/parser-4.5.0

chore(deps): bump @typescript-eslint/parser from 4.4.1 to 4.5.0

AndrewLeedham committed:
Merge pull request #431 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-config-prettier-6.14.0

chore(deps): bump eslint-config-prettier from 6.13.0 to 6.14.0

dependabot[bot] committed:
chore(deps-dev): bump @auto-it/slack from 9.59.0 to 9.60.0

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 9.59.0 to 9.60.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.59.0...v9.60.0)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #434 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-9.60.0

chore(deps-dev): bump @auto-it/slack from 9.59.0 to 9.60.0

dependabot[bot] committed:
chore(deps): bump eslint-plugin-jsdoc from 30.6.5 to 30.7.3

Bumps [eslint-plugin-jsdoc](https://github.com/gajus/eslint-plugin-jsdoc) from 30.6.5 to 30.7.3.
- [Release notes](https://github.com/gajus/eslint-plugin-jsdoc/releases)
- [Commits](https://github.com/gajus/eslint-plugin-jsdoc/compare/v30.6.5...v30.7.3)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #436 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-plugin-jsdoc-30.7.3

chore(deps): bump eslint-plugin-jsdoc from 30.6.5 to 30.7.3

dependabot[bot] committed:
chore(deps-dev): bump jest from 26.6.0 to 26.6.1

Bumps [jest](https://github.com/facebook/jest) from 26.6.0 to 26.6.1.
- [Release notes](https://github.com/facebook/jest/releases)
- [Changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md)
- [Commits](https://github.com/facebook/jest/compare/v26.6.0...v26.6.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump @auto-it/slack from 9.60.0 to 9.60.1

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 9.60.0 to 9.60.1.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.60.0...v9.60.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump eslint from 7.11.0 to 7.12.1

Bumps [eslint](https://github.com/eslint/eslint) from 7.11.0 to 7.12.1.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v7.11.0...v7.12.1)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #442 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-7.12.1

chore(deps-dev): bump eslint from 7.11.0 to 7.12.1

AndrewLeedham committed:
Merge pull request #440 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-9.60.1

chore(deps-dev): bump @auto-it/slack from 9.60.0 to 9.60.1

AndrewLeedham committed:
Merge pull request #438 from xeroxinteractive/dependabot/npm_and_yarn/next/jest-26.6.1

chore(deps-dev): bump jest from 26.6.0 to 26.6.1

dependabot[bot] committed:
chore(deps-dev): bump auto from 9.60.0 to 9.60.1

Bumps [auto](https://github.com/intuit/auto) from 9.60.0 to 9.60.1.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.60.0...v9.60.1)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #441 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-9.60.1

chore(deps-dev): bump auto from 9.60.0 to 9.60.1

dependabot[bot] committed:
chore(deps-dev): bump typescript from 4.0.3 to 4.0.5

Bumps [typescript](https://github.com/Microsoft/TypeScript) from 4.0.3 to 4.0.5.
- [Release notes](https://github.com/Microsoft/TypeScript/releases)
- [Commits](https://github.com/Microsoft/TypeScript/compare/v4.0.3...v4.0.5)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump jest-package-audit from 3.1.2 to 3.2.0

Bumps [jest-package-audit](https://github.com/xeroxinteractive/jest-package-audit) from 3.1.2 to 3.2.0.
- [Release notes](https://github.com/xeroxinteractive/jest-package-audit/releases)
- [Changelog](https://github.com/xeroxinteractive/jest-package-audit/blob/v3.2.0/CHANGELOG.md)
- [Commits](https://github.com/xeroxinteractive/jest-package-audit/compare/v3.1.2...v3.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump auto from 9.60.1 to 9.61.0

Bumps [auto](https://github.com/intuit/auto) from 9.60.1 to 9.61.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.60.1...v9.61.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump @types/react from 16.9.53 to 16.9.54

Bumps [@types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react) from 16.9.53 to 16.9.54.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #443 from xeroxinteractive/dependabot/npm_and_yarn/next/typescript-4.0.5

chore(deps-dev): bump typescript from 4.0.3 to 4.0.5

AndrewLeedham committed:
Merge pull request #444 from xeroxinteractive/dependabot/npm_and_yarn/next/jest-package-audit-3.2.0

chore(deps-dev): bump jest-package-audit from 3.1.2 to 3.2.0

AndrewLeedham committed:
Merge pull request #445 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-9.61.0

chore(deps-dev): bump auto from 9.60.1 to 9.61.0

AndrewLeedham committed:
Merge pull request #446 from xeroxinteractive/dependabot/npm_and_yarn/next/types/react-16.9.54

chore(deps-dev): bump @types/react from 16.9.53 to 16.9.54

dependabot[bot] committed:
chore(deps-dev): bump @auto-it/slack from 9.60.1 to 9.61.0

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 9.60.1 to 9.61.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.60.1...v9.61.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump @types/react from 16.9.54 to 16.9.55

Bumps [@types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react) from 16.9.54 to 16.9.55.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump @typescript-eslint/eslint-plugin from 4.5.0 to 4.6.0

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 4.5.0 to 4.6.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.6.0/packages/eslint-plugin)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump @typescript-eslint/parser from 4.5.0 to 4.6.0

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 4.5.0 to 4.6.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.6.0/packages/parser)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #447 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-9.61.0

chore(deps-dev): bump @auto-it/slack from 9.60.1 to 9.61.0

AndrewLeedham committed:
Merge pull request #448 from xeroxinteractive/dependabot/npm_and_yarn/next/types/react-16.9.55

chore(deps-dev): bump @types/react from 16.9.54 to 16.9.55

AndrewLeedham committed:
Merge pull request #449 from xeroxinteractive/dependabot/npm_and_yarn/next/typescript-eslint/eslint-plugin-4.6.0

chore(deps): bump @typescript-eslint/eslint-plugin from 4.5.0 to 4.6.0

AndrewLeedham committed:
Merge pull request #450 from xeroxinteractive/dependabot/npm_and_yarn/next/typescript-eslint/parser-4.6.0

chore(deps): bump @typescript-eslint/parser from 4.5.0 to 4.6.0

dependabot[bot] committed:
chore(deps-dev): bump auto from 9.61.0 to 10.0.2

Bumps [auto](https://github.com/intuit/auto) from 9.61.0 to 10.0.2.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.61.0...v10.0.2)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump @auto-it/slack from 9.61.0 to 10.0.2

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 9.61.0 to 10.0.2.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v9.61.0...v10.0.2)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump eslint-config-prettier from 6.14.0 to 6.15.0

Bumps [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) from 6.14.0 to 6.15.0.
- [Release notes](https://github.com/prettier/eslint-config-prettier/releases)
- [Changelog](https://github.com/prettier/eslint-config-prettier/blob/master/CHANGELOG.md)
- [Commits](https://github.com/prettier/eslint-config-prettier/compare/v6.14.0...v6.15.0)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #453 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-config-prettier-6.15.0

chore(deps): bump eslint-config-prettier from 6.14.0 to 6.15.0

AndrewLeedham committed:
Merge pull request #451 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-10.0.2

chore(deps-dev): bump auto from 9.61.0 to 10.0.2

AndrewLeedham committed:
Merge pull request #452 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-10.0.2

chore(deps-dev): bump @auto-it/slack from 9.61.0 to 10.0.2

dependabot[bot] committed:
chore(deps): bump eslint-plugin-jsdoc from 30.7.3 to 30.7.6

Bumps [eslint-plugin-jsdoc](https://github.com/gajus/eslint-plugin-jsdoc) from 30.7.3 to 30.7.6.
- [Release notes](https://github.com/gajus/eslint-plugin-jsdoc/releases)
- [Commits](https://github.com/gajus/eslint-plugin-jsdoc/compare/v30.7.3...v30.7.6)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #454 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-plugin-jsdoc-30.7.6

chore(deps): bump eslint-plugin-jsdoc from 30.7.3 to 30.7.6

dependabot[bot] committed:
chore(deps): bump @typescript-eslint/parser from 4.6.0 to 4.6.1

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 4.6.0 to 4.6.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.6.1/packages/parser)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump jest from 26.6.1 to 26.6.2

Bumps [jest](https://github.com/facebook/jest) from 26.6.1 to 26.6.2.
- [Release notes](https://github.com/facebook/jest/releases)
- [Changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md)
- [Commits](https://github.com/facebook/jest/compare/v26.6.1...v26.6.2)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump auto from 10.0.2 to 10.2.0

Bumps [auto](https://github.com/intuit/auto) from 10.0.2 to 10.2.0.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v10.0.2...v10.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump @typescript-eslint/eslint-plugin from 4.6.0 to 4.6.1

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 4.6.0 to 4.6.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.6.1/packages/eslint-plugin)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #458 from xeroxinteractive/dependabot/npm_and_yarn/next/typescript-eslint/eslint-plugin-4.6.1

chore(deps): bump @typescript-eslint/eslint-plugin from 4.6.0 to 4.6.1

AndrewLeedham committed:
Merge pull request #456 from xeroxinteractive/dependabot/npm_and_yarn/next/jest-26.6.2

chore(deps-dev): bump jest from 26.6.1 to 26.6.2

AndrewLeedham committed:
Merge pull request #455 from xeroxinteractive/dependabot/npm_and_yarn/next/typescript-eslint/parser-4.6.1

chore(deps): bump @typescript-eslint/parser from 4.6.0 to 4.6.1

AndrewLeedham committed:
Merge pull request #457 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-10.2.0

chore(deps-dev): bump auto from 10.0.2 to 10.2.0

tinytim84 committed:
fix: package versions

tinytim84 committed:
fix: update auto

tinytim84 committed:
fix: move git fetch to workflow

tinytim84 committed:
fix: remove plugin

tinytim84 committed:
Merge branch 'release' into next

dependabot[bot] committed:
chore(deps-dev): bump @auto-it/slack from 10.0.2 to 10.2.1

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 10.0.2 to 10.2.1.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v10.0.2...v10.2.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump jest from 26.6.2 to 26.6.3

Bumps [jest](https://github.com/facebook/jest) from 26.6.2 to 26.6.3.
- [Release notes](https://github.com/facebook/jest/releases)
- [Changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md)
- [Commits](https://github.com/facebook/jest/compare/v26.6.2...v26.6.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump browserslist-adobe-analytics from 1.0.7 to 1.1.0

Bumps [browserslist-adobe-analytics](https://github.com/xeroxinteractive/browserslist-adobe-analytics) from 1.0.7 to 1.1.0.
- [Release notes](https://github.com/xeroxinteractive/browserslist-adobe-analytics/releases)
- [Changelog](https://github.com/xeroxinteractive/browserslist-adobe-analytics/blob/v1.1.0/CHANGELOG.md)
- [Commits](https://github.com/xeroxinteractive/browserslist-adobe-analytics/compare/v1.0.7...v1.1.0)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #463 from xeroxinteractive/dependabot/npm_and_yarn/next/jest-26.6.3

chore(deps-dev): bump jest from 26.6.2 to 26.6.3

dependabot[bot] committed:
chore(deps-dev): bump auto from 10.2.0 to 10.2.1

Bumps [auto](https://github.com/intuit/auto) from 10.2.0 to 10.2.1.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v10.2.0...v10.2.1)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #462 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-10.2.1

chore(deps-dev): bump @auto-it/slack from 10.0.2 to 10.2.1

AndrewLeedham committed:
Merge pull request #464 from xeroxinteractive/dependabot/npm_and_yarn/next/browserslist-adobe-analytics-1.1.0

chore(deps-dev): bump browserslist-adobe-analytics from 1.0.7 to 1.1.0

AndrewLeedham committed:
Merge pull request #461 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-10.2.1

chore(deps-dev): bump auto from 10.2.0 to 10.2.1

tinytim84 committed:
chore: update labels and auto

dependabot[bot] committed:
chore(deps-dev): bump auto from 10.2.1 to 10.2.3

Bumps [auto](https://github.com/intuit/auto) from 10.2.1 to 10.2.3.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v10.2.1...v10.2.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump eslint-plugin-jsdoc from 30.7.6 to 30.7.7

Bumps [eslint-plugin-jsdoc](https://github.com/gajus/eslint-plugin-jsdoc) from 30.7.6 to 30.7.7.
- [Release notes](https://github.com/gajus/eslint-plugin-jsdoc/releases)
- [Commits](https://github.com/gajus/eslint-plugin-jsdoc/compare/v30.7.6...v30.7.7)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham commented on .github/labels.yml:
```diff
@@ -29,30 +29,33 @@
-  description: Changes only affect the internal API
-  color: 696969
-- name: documentation
+- name: 'bump: feature'
```
> Maybe there is a better prefix that encompasses both cases? - me

I meant for the chore ones and these.
```suggestion
- name: 'change: feature'
```

tinytim84 replied:
Sorry, I misunderstood.  I've changed them all to `change`.  I've made the label colour different for the two that don't make a release.

AndrewLeedham committed:
Merge pull request #465 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-10.2.3

chore(deps-dev): bump auto from 10.2.1 to 10.2.3

AndrewLeedham committed:
Merge pull request #466 from xeroxinteractive/dependabot/npm_and_yarn/next/eslint-plugin-jsdoc-30.7.7

chore(deps): bump eslint-plugin-jsdoc from 30.7.6 to 30.7.7

dependabot[bot] committed:
chore(deps-dev): bump @auto-it/slack from 10.2.1 to 10.2.3

Bumps [@auto-it/slack](https://github.com/intuit/auto) from 10.2.1 to 10.2.3.
- [Release notes](https://github.com/intuit/auto/releases)
- [Changelog](https://github.com/intuit/auto/blob/master/CHANGELOG.md)
- [Commits](https://github.com/intuit/auto/compare/v10.2.1...v10.2.3)

Signed-off-by: dependabot[bot] <support@github.com>

AndrewLeedham committed:
Merge pull request #467 from xeroxinteractive/dependabot/npm_and_yarn/next/auto-it/slack-10.2.3

chore(deps-dev): bump @auto-it/slack from 10.2.1 to 10.2.3

tinytim84 committed:
chore: update labels

AndrewLeedham commented on .github/labels.yml:
```diff
@@ -29,28 +29,28 @@
   description: Updates to dependancies only
   color: fcf3b8
-- name: 'change: test'
+- name: 'change: tests'
```
The `package.json` version is not pluralised?
https://github.com/xeroxinteractive/config/pull/460/files/e141540a9f3a123fa3af801490ce7bff248b76a8..17daddaaf9d4677d2a67c1761b9a277bce1d1ff8#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R116

tinytim84 replied:
Changed.

tinytim84 committed:
chore: update label
